### PR TITLE
new feature template

### PR DESCRIPTION
**Pull request by runleonarun**
## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [ ] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-dbt-feature.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        * Before you file an issue read the [Contributing guide](https://github.com/dbt-labs/docs.getdbt.com#contributing).
+        * Check to make sure someone hasn't already opened a similar [issue](https://github.com/dbt-labs/docs.getdbt.com/issues).
+
+  - type: checkboxes
+    id: contributions
+    attributes:
+      label: Contributions
+      description: This applies to new, unreleased content.
+      options:
+        - label: I am a Product manager at dbt who is responsible for this feature.
```
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/new-dbt-feature.yml]:
```suggestion
        - label: I am a PM at dbt who is responsible for this feature. 
```

File: .github/ISSUE_TEMPLATE/new-dbt-feature.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    attributes:
+      label: Where does this content belong?
+      description: |
+        - Give as much detail as you can to help us understand where you expect the content to live.  
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Why this feature? Why now?
```
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/new-dbt-feature.yml]:
```suggestion
      label: Link to source material
```

File: .github/ISSUE_TEMPLATE/new-dbt-feature.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      description: |
+        - Add any other context or screenshots about the feature here.
```
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/new-dbt-feature.yml]:
```suggestion
        Use the [source material template](https://docs.google.com/document/d/1lLWGMXJFjkY4p7r8ZKhBX73dOLmIjgXZBYq39LqmAJs/edit) to provide source material for this feature. 
```

File: .github/ISSUE_TEMPLATE/new-dbt-feature.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        - How will this feature be helpful?
```
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/new-dbt-feature.yml]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/new-dbt-feature.yml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        - What is the expected outcome?
```
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/new-dbt-feature.yml]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/new-dbt-feature.yml [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        * Before you file an issue read the [Contributing guide](https://github.com/dbt-labs/docs.getdbt.com#contributing).
+        * Check to make sure someone hasn't already opened a similar [issue](https://github.com/dbt-labs/docs.getdbt.com/issues).
+
+  - type: checkboxes
+    id: contributions
+    attributes:
+      label: Contributions
+      description: This applies to new, unreleased content.
+      options:
+        - label: I am a PM at dbt who is responsible for this feature.
```
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/new-dbt-feature.yml]:
```suggestion
        - label: I am a PM or subject matter expert at dbt who is responsible for this feature. 
```

File: .github/ISSUE_TEMPLATE/new-dbt-feature.yml [hunk 7]
```diff
@@ -0,0 +1,34 @@
+name: New product feature
+description: Make a suggestion to add docs for a new dbt product feature.
+labels: ["content","new dbt feature"]
```
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/new-dbt-feature.yml]:
```suggestion
labels: ["content","upcoming release"]
```

File: .github/ISSUE_TEMPLATE/new-dbt-feature.yml [hunk 8]
```diff
@@ -0,0 +1,34 @@
+name: New product feature
+description: Make a suggestion to add docs for a new dbt product feature.
```
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/new-dbt-feature.yml]:
```suggestion
description: For dbt PMs to add docs for their new or updated dbt product features.
```

File: .github/ISSUE_TEMPLATE/new-dbt-feature.yml [hunk 9]
```diff
@@ -0,0 +1,34 @@
+name: New product feature
```
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/new-dbt-feature.yml]:
```suggestion
name: Start docs process
```

File: .github/ISSUE_TEMPLATE/new-dbt-feature.yml [hunk 10]
```diff
@@ -0,0 +1,34 @@
+name: Start docs process
```
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/new-dbt-feature.yml]:
```suggestion
name: Start docs project for a new feature
```

File: .github/ISSUE_TEMPLATE/new-dbt-feature.yml [hunk 11]
```diff
@@ -0,0 +1,34 @@
+name: Start docs project for a new feature
+description: For dbt PMs to add docs for their new or updated dbt product features.
+labels: ["content","upcoming release"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        * You can ask questions or submit ideas for the dbt docs in [Discussions](https://github.com/dbt-labs/docs.getdbt.com/discussions)
```
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/new-dbt-feature.yml]:
```suggestion
```

**Review by jasnonaz — APPROVED** [review]
Looks great!

**Pull request merged**